windows/src/main/csharp/ch/cyberduck/ui/Views/TransfersWindow.xaml

```diff
@@ -0,0 +1,373 @@
+                    <StackPanel>
+                        <Image Source="{xaml:Icon CleanAll,Size=24}"
+                               Height="24"
+                               Width="24"
+                               Style="{StaticResource ButtonImage}" />
+                        <TextBlock Name="CleanAllButtonLabel"
+                                   Text="{xaml:Text 'Clean Up',Table=Localizable}" />
+                    </StackPanel>
+                </Button>
+                <Separator Visibility="{Binding ToolbarSeparatorLeft,Converter={StaticResource b2vis}}" />
+                <Button Command="{Binding ResumeCommand}"
+                        Visibility="{Binding ToolbarResume,Converter={StaticResource b2vis}}"
+                        Margin="3,0"
+                        AutomationProperties.LabeledBy="{Binding ElementName=ResumeButtonLabel}">
+                    <StackPanel>
+                        <Image Source="{xaml:Icon Resume,Size=24}"
+                               Height="24"
+                               Width="24"
+                               Style="{StaticResource ButtonImage}" />
+                        <TextBlock Name="ResumeButtonLabel"
+                                   Text="{xaml:Text Resume,Table=Transfer}" />
+                    </StackPanel>
+                </Button>
+                <Button Command="{Binding StopCommand}"
+                        Visibility="{Binding ToolbarStop,Converter={StaticResource b2vis}}"
+                        Margin="3,0"
+                        AutomationProperties.LabeledBy="{Binding ElementName=StopButtonLabel}">
+                    <StackPanel>
+                        <Image Source="{xaml:Icon Stop,Size=24}"
+                               Height="24"
+                               Width="24"
+                               Style="{StaticResource ButtonImage}" />
+                        <TextBlock Name="StopButtonLabel"
+                                   Text="{xaml:Text Stop,Table=Localizable}" />
+                    </StackPanel>
+                </Button>
+                <Button Command="{Binding ReloadCommand}"
+                        Visibility="{Binding ToolbarReload,Converter={StaticResource b2vis}}"
+                        Margin="3,0"
+                        AutomationProperties.LabeledBy="{Binding ElementName=ReloadButtonLabel}">
+                    <StackPanel>
+                        <Image Source="{xaml:Icon Reload,Size=24}"
+                               Height="24"
+                               Width="24"
+                               Style="{StaticResource ButtonImage}" />
+                        <TextBlock Name="ReloadButtonLabel"
+                                   Text="{xaml:Text Reload,Table=Localizable}" />
+                    </StackPanel>
+                </Button>
+            </StackPanel>
+
+            <StackPanel Orientation="Horizontal"
```

Comment:
WPF toolbar doesn't allow start/end alignment of elements. So use two StackPanels docked left/right.